### ISSUE TITLE
"There was an issue while running the post deploy script"

### ISSUE THREAD

**Issue by BooDoo** [BooDoo/ebooks_example#1]
Cannot currently "Deploy to Heroku" through web workflow; receive following error at "Running Scripts & Scaling Dynos" step:

```
There was an issue while running the post deploy script: 
[empty text box]
```

Investigating…

First fix attempt: removing both `scripts.postdeply` and `addons` from `app.json`

**Comment by BooDoo** [comment]
Still no dice after removing those fields from `app.json`. 

Current config deploys & works fine from heroku cli using a pre-filled `secret.env`:

``` bash
heroku create cli-ebooks && \
heroku config:push --env=secret.env && \
git push heroku master && \
heroku ps:scale worker=1
```

but cryptic error through web deploy dashboard. Will follow-up with Heroku support team tmw.

**Comment by grampajoe** [comment]
After waiting for the "Running scripts & scaling dynos" step for a while, I saw:

![screen shot 2015-01-07 at 5 59 39 pm](https://cloud.githubusercontent.com/assets/55005/5655004/19ad2802-9697-11e4-94c6-ba92257d8979.png)

**Comment by grampajoe** [comment]
The app was present on my Heroku dashboard after seeing that error, and I scaled it up to 1 dyno. Based on the logs, it downloaded the archive and it should be online, but I haven't seen tweets yet.

**Comment by BooDoo** [comment]
try DMing it with "tweet" from the "source" account and see if it does?

**Comment by BooDoo** [comment]
Yeah. So I changed nothing and now it's magically deploying cleanly for me. Weird. 

Keeping open to monitor more attempts over the next day or so.

**Comment by grampajoe** [comment]
I tried DMing it, and it doesn't seem to have picked it up. I'll rebuild it with the current config, although I do think removing that postdeploy script should help.

**Comment by grampajoe** [comment]
Deploying the new config (bb22671630782ab410da4ad8cf545a853f4a1474) worked fine. There was some delay before my bot started tweeting, but I guess that could just be how it's configured?

**Comment by BooDoo** [comment]
Closing as resolved by bb22671, at least unless/until this weirdness rears its head again.
